3.5/release-notes-known-issues35.md

```diff
@@ -33,6 +33,13 @@ Upgrading
 | **Date Added:** 2019-05-16 <br> **Component:** ArangoSearch <br> **Deployment Mode:** All <br> **Description:** Upgrading ArangoSearch Views from 3.4.x to 3.5.0 pre-releases is not supported for the time being. You may recreate them manually however. If the server encounters views in an upgrade, it will fail with the following message: <br>`FATAL [7db7c] {arangosearch} Upgrading views is not supported in 3.5RC1, please drop all the existing views and manually recreate them after the upgrade is complete` <br> **Affected Versions:** 3.5.0-RC.x <br> **Fixed in Versions:** - <br> **Reference:** N/A |
 | **Date Added:** 2019-05-16 <br> **Component:** arangod <br> **Deployment Mode:** All <br> **Description:** Bugfix release upgrades such as 3.4.4 to 3.4.5 may not create a backup of the database directory even if they should. Please create a copy manually before upgrading. <br> **Affected Versions:** 3.4.x (Windows and Linux) <br> **Fixed in Versions:** - <br> **Reference:** [arangodb/planning#3745](https://github.com/arangodb/planning/issues/3745){:target="_blank"} (internal) |
 
+Stream Transactions
+-------------------
+| Issue      |
+|------------|
+| **Date Added:** 2019-08-19 <br> **Component:** Transactions <br> **Deployment Mode:** All <br> **Description:** Stream Transactions currently do not honor the limits described in the documentation. Currently the idle timeout of 10 seconds will not be enforced, neither will the maximum size of transaction be enforced. It is planned to add the enforcements of the limits with ArangoDB 3.5.1. <br> **Affected Versions:** 3.4.0RC1 to 3.4.0 (including) <br> **Fixed in Versions:** - <br> **Reference:** N/A |
+| **Date Added:** 2019-08-19 <br> **Component:** Transactions <br> **Deployment Mode:** All <br> **Description:** Stream Transactions currently do not support the graph operations that are initiated via the `general-graph` JavaScript module or via the REST API at `/_api/gharial`. Right now these operations will act as if no stream transaction is present. Stream transaction support for these operations will be added later. <br> **Affected Versions:** 3.4.0RC1 to 3.4.0 (including) <br> **Fixed in Versions:** - <br> **Reference:** N/A |
```

Comment:
Is the affected version correct @jsteemann

Reply:
Fixed.